File: 1413. Minimum Value to Get Positive Step by Step Sum/solution.cpp
class Solution {
public:
    int minStartValue(vector<int>& nums) {
        int _min = nums[0];
        for (int i = 1; i < nums.size(); i++) {
            nums[i] += nums[i-1];
            _min = min(_min, nums[i]);
        }
        if (_min < 1) return 1 - _min;
        return 1;
    }
};

File: 1413. Minimum Value to Get Positive Step by Step Sum/solution.py
def minStartValue(self, nums: List[int]) -> int:
    
    _min = nums[0]
    for i in range(1, len(nums)):
        nums[i] += nums[i-1]
        _min = min(_min, nums[i])
        
    return 1 - _min if _min < 1 else 1